Add `genproto` error troubleshoot step for manual bumps (#18431)
etcd-io/etcd: Documentation/contributor-guide/dependency_management.md

When you bump multiple dependencies in one PR, it's recommended to create a separate commit for each dependency. But it isn't a must; for example,
you can get all dependencies bumping for the module `go.etcd.io/etcd/tools/v3` included in one commit.

> #### Troubleshooting 

Reviewer: Nit: Markdownlint suggests blank lines surround headers.
```suggestion
#### Troubleshooting 

```

> In an event of bumping `grpc-*` dependencies, it might change `*.proto` file which can result in the following error:

Reviewer 2: ```suggestion
In an event of bumping the version of protoc, protoc plugins or grpc-gateway, it might change `*.proto` file which can result in the following error:
```

> ```bash

Reviewer: Likewise, blank lines surround fenced code blocks.
```suggestion

```bash
```

[0;31mFAIL: 'genproto' FAILED at Wed Jul 31 07:09:08 UTC 2024
make: *** [Makefile:134: verify-genproto] Error 255

> ```
> To fix the above error, run the following script from the root of etcd repository:
> ```bash

Reviewer: ```suggestion

To fix the above error, run the following script from the root of etcd repository:

```bash
```
